Makefile

```diff
@@ -50,7 +50,7 @@ test-not-yelpy: .paasta/bin/activate
 	.paasta/bin/tox -e tests
 
 quick-test: .tox/py38-linux
-	TZ=UTC .tox/py38-linux/bin/py.test --last-failed -x -- tests
+	TZ=UTC .tox/py38-linux/bin/py.test --failed-first -x --disable-warnings -- tests
```

Comment:
I use `make quick-test` for iterative test fixing, and `--last-failed` would exit after successfully running one test that had previously failed. With `--failed-first`, if the previously-failing test succeeds, it will continue on to run all the other tests.

Without `--disable-warnings`, we have several pages of warnings (mostly from libraries we import) that mean you need to scroll up to see the traceback of the failed test.